subsys/net/ip/net_context.c

```diff
@@ -397,6 +425,17 @@ static int bind_default(struct net_context *context)
 	}
 #endif
 
+	if (IS_ENABLED(CONFIG_NET_SOCKETS_PACKET) && family == AF_PACKET) {
+		struct sockaddr_ll ll_addr;
+
+		ll_addr.sll_family = AF_PACKET;
+		ll_addr.sll_protocol = ETH_P_ALL;
+		ll_addr.sll_ifindex = net_if_get_by_iface(net_if_get_default());
```

Comment:
At some point in future, we might want to only use an interface that supports AF_PACKET. For that there would probably be a capability flag in net_if that would indicate the AF_PACKET support.

Reply:
ok.